lib/variable/comparison.cpp

```diff
@@ -18,16 +18,31 @@ namespace scipp::variable {
 
 namespace {
 Variable _values(Variable &&in) { return in.has_variances() ? values(in) : in; }
+
+template <class T, class... Remaining>
+std::optional<Variable>
+try_isclose_spatial(const Variable &a, const Variable &b, const Variable &rtol,
+                    const Variable &atol, const NanComparisons equal_nans) {
+  if (a.dtype() == dtype<T>)
+    return std::optional(
+        all(isclose(a.elements<T>(), b.elements<T>(), rtol, atol, equal_nans),
+            Dim::InternalStructureComponent));
```

Comment:
We should probably document clearly (in the Python docstring) that this does element-wise comparison and does not, e.g., consider closeness of equivalent rotations.

Reply:
Good point! Now I am unsure if we should even support it. Is it useful outside of unit tests?

Reply:
Not sure.